TradeItIosTicketSDK2/TradeItYahooTradingTicketViewController.swift

```diff
@@ -1,13 +1,14 @@
 import UIKit
 import MBProgressHUD
 
-class TradeItYahooTradingTicketViewController: CloseableViewController, UITableViewDelegate, UITableViewDataSource {
+class TradeItYahooTradingTicketViewController: CloseableViewController, UITableViewDelegate, UITableViewDataSource, TradeItYahooAccountSelectionViewControllerDelegate {
     @IBOutlet weak var tableView: TradeItYahooTradingTicketTableView!
     @IBOutlet weak var reviewOrderButton: UIButton!
 
     var alertManager = TradeItAlertManager()
     let viewProvider = TradeItViewControllerProvider(storyboardName: "TradeItYahoo")
     var selectionViewController: TradeItSelectionViewController!
+    var accountSelectionViewController: TradeItYahooAccountSelectionViewController!
```

Comment:
Maybe? I'd say probably not in this case because the trading ticket screen isn't actually relinquishing flow back to the UI screen, it's just going off on a little side-quest to pick a different account.